_data/plugins/katello.yml

```diff
@@ -1,4 +1,6 @@
 versions:
+  - name: "3.16"
+    display: "3.16"
```

Comment:
The display should be 'RC1' here. That will show in the dropdown for the version selector